packages/workflows/src/handlers/product/list-products.ts

```diff
@@ -1,17 +1,20 @@
 import { ProductTypes, WorkflowTypes } from "@medusajs/types"
 import { WorkflowArguments } from "../../helper"
 
+type HandlerInput = {
+  productIds: string[]
```

Comment:
suggestion: `ids` should be fine, the handler name speaks for itself
```suggestion
  ids: string[]
```